2022-07_sprint/schedule_table.md

```diff
@@ -3,6 +3,8 @@
 <meta name="twitter:description" content="Beginner friendly working sessions to get started contributing to PyMC">
 <meta name="twitter:image" content="https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/_static/banner_2022_07/2022_07_banner_1280×640px.png">
 
+#
```

Comment:
This is probably here because sphinx was complaining about the "increase" from level 0 to level 2 header. But that isn't really happening. This file is not a page, but common content between two pages, each with its own level 1 header, so we need this to start at level 2 heading.

If it is complaining, it is because sphinx is interpreting this file also as a standalone page (just checked and yes, the page exists even if it is not reachable because no links point to it: https://pymc-data-umbrella--100.org.readthedocs.build/en/100/2022-07_sprint/schedule_table.html). I think the right way to fix the warning is excluding this file from the sphinx rendering. This is done from `conf.py`. 

One option is adding the path to this file explicitly. However, I think using the include directive to have single source of truth for content that we want to show in multiple pages is very useful, so we could add `*.part.md` to the exclude patterns in conf.py and update the filename (and use it if we do this in other places).

The place to add this is `exclude_patterns` variable: https://github.com/pymc-devs/pymc-data-umbrella/blob/main/conf.py#L77

Reply:
Side note, this option also adds this spurious empty heading (visible only on hover because all headings even if empty have an id with the `#` so we can link to specific sections within a page)

![imatge](https://user-images.githubusercontent.com/23738400/174814660-274a1c8b-0ce0-4626-b6ff-36b494b1387f.png)